ml-agents/mlagents/trainers/trainer/rl_trainer.py

```diff
@@ -256,6 +258,19 @@ def _maybe_save_model(self, step_after_process: int) -> None:
         if step_after_process >= self._next_save_step and self.get_step != 0:
             self._checkpoint()
 
+    def _warn_if_group_reward(self, buffer: AgentBuffer) -> None:
+        """
+        Warn if the trainer receives a Group Reward but isn't a multiagent trainer (e.g. POCA).
+        """
+        if not self._has_warned_group_rewards:
+            group_reward = np.sum(buffer[BufferKey.GROUP_REWARD])
+            if group_reward > 0.0:
```

Comment:
Will there be floating point issues ?
I think we should do something like 
```suggestion
            if group_reward > self.EPSILON: # No not use this suggestion, there is no static EPSILON in RLTrainer
```

Reply:
There shouldn't be, as an unassigned group reward is exactly 0.0